### PR TITLE
[Growth] Reminder email implementation

### PR DESCRIPTION
**Pull request by franckc**
First pull request? Read our [guide to contributing](http://docs.originprotocol.com/#contributing)

### Description:

Implemented /inviteRemind mutation.

### Checklist:

- [X] Test your work and double-check to confirm that you didn't break anything
- [ ] Wrap any displayed ETH addresses with [`formattedAddress`](https://github.com/OriginProtocol/origin/blob/master/origin-dapp/src/utils/user.js#L15-L17)
- [ ] Wrap any new text/strings for translation
- [ ] Run `npm run translations` if there are any changes to translated strings
- [ ] Map any new environment variables with a default value in the Webpack config
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/origin-docs)

### CODE REVIEW

File: origin-growth/src/resources/email.js [hunk 1]
```diff
@@ -81,8 +114,7 @@ async function sendInvites(referrer, recipients) {
     }
 
     // Send the invite code to the recipient.
-    const targetUrl = `${dappUrl}/#/welcome/${code}`
-    const { subject, text, html } = generateInviteEmail(referrerName, targetUrl)
+    const { subject, text, html } = generateEmail(referrerName, targetUrl)
```
Comment (sparrowDom) [inline origin-growth/src/resources/email.js]: looks like this should be:
```
const { subject, text, html } = generateEmail('invite', referrerName, targetUrl)
```

File: origin-growth/src/resources/email.js [hunk 2]
```diff
@@ -118,4 +150,48 @@ async function sendInvites(referrer, recipients) {
...
+    logger.error('Reminder sending only supports email for now.')
+    return
+  }
+
+  logger.info(`Sending reminder email on behalf od ${referrer}`)
+  const { referrerName, targetUrl } = await _getEmailVars(referrer)
+  const recipient = invite.refereeContact
+
+  // Send the reminder email.
+  const { subject, text, html } = generateEmail(referrerName, targetUrl)
```
Comment (sparrowDom) [inline origin-growth/src/resources/email.js]: and this:
```
const { subject, text, html } = generateEmail('remind', referrerName, targetUrl)
```

File: origin-growth/src/resources/email.js (hunk 1 above)
Comment (franckc) [reply origin-growth/src/resources/email.js]: ah yeah. good catch ! thank you for being such a thorough reviewer of my bad code :)

Comment (sparrowDom) [reply origin-growth/src/resources/email.js]: Well you catch a lot of bugs in my code... Saves a lot of pain with debugging down the line

**Pull request merged**